docs/doc/11-integrations/00-api/03-streaming-load.md

```diff
@@ -18,17 +18,13 @@ curl -H "insert_sql:<value>" -F "upload=@<file_location>" [-F "upload=@<file_loc
 ```
 ## Explaining Argument `-H`
 
-The request usually includes many occurrences of the argument `-H` and each is followed by one of the following parameters to tell Databend how to handle the file you're loading data from. Please note that `insert_sql` is required.
+The request usually includes many occurrences of the argument `-H` and each is followed by one of the following parameters to tell Databend how to handle the file you're loading data from. Please note that `insert_sql` is required. For the parameter `FILE_FORMAT` in `insert_sql`, see [Input File Formats](../30-file-format-options.md).
```

Comment:
1. `-H` is a general option of curl to pass HTTP header,  current desc seems like it databend special, and is not Professional. 
2.  we only use  `insert_sql` header
3. FILE_FORMAT cause is used in the SQL

 I  curently we only use `insert_sql` header to pass the insert sql with FILE_FORMAT.
for curl, just used `-H`, like ...
for options of FILE_FORMAT refer to ..
```

Reply:
cc @Xuanwo  @BohuTANG

Reply:
I didnt know that -H is part of CURL. I will re-edit it.